pwa/src/templates/templateParts/cardsResultsTemplate/CardsResultsTemplate.tsx

```diff
@@ -26,7 +26,7 @@ export const CardsResultsTemplate: React.FC<CardsResultsTemplateProps> = ({ requ
                 {request.Publicatiedatum ? translateDate(i18n.language, request.Publicatiedatum) : "N.v.t."}
               </CardHeaderDate>
               <CardHeaderTitle className={styles.title}>
-                <Heading2>{request.Titel !== "" ? request.Titel : "Geen titel beschikbaar"}</Heading2>
+                <Heading2>{request.Titel !== ("" || null) ? request.Titel : "Geen titel beschikbaar"}</Heading2>

```

Comment:
I think `request.Titel ?? "Geen titel beschikbaar"` will do the same. 

**[Nullish coalescing operator.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)**